android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -42,13 +50,10 @@
 import android.graphics.Rect;
 import android.net.Uri;
 import android.os.Build;
-import android.view.MotionEvent;
-import android.view.View;
 import android.webkit.WebSettings;
 import android.webkit.WebView;
 
-@SuppressWarnings("deprecation")
-public class TiUIWebView extends TiUIView
+@SuppressWarnings("deprecation") public class TiUIWebView extends TiUIView
```

Comment:
I'm not sure you've imported it and used it properly then. The settings include this:
```xml
<setting id="org.eclipse.jdt.core.formatter.insert_new_line_after_annotation_on_method" value="insert"/>
```